lib/utils.ts

```diff
@@ -140,13 +140,13 @@ export function num2hex(num: number): string {
  */
 export function getVersion(): string {
 	if (!VERSION) {
-		let revision = ''
 		try {
-			revision = execSync('git rev-parse --short HEAD').toString().trim()
-		} catch (error) {
-			// git not installed
+			const shellCmd = 'command -v git || exit 0; git rev-parse --short HEAD'
+			const revision = execSync(shellCmd).toString().trim()
```

Comment:
Having the shell command and this code on the same line, seemed to make it too long/complex.  So, I split them up, if that's an issue, I can put them back together.